panel_client/perfect_panel.py

```diff
@@ -0,0 +1,103 @@
+"""Demonstration/debugging panel. Uses the shell for I/O."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import Queue
+import threading
+from panel_client import PanelStateBase
+
+CONTROL_SCHEMES = [
+    [
```

Comment:
Can you reproduce the schema docs from `panel.py` here? Since this code is going to replace `panel.py` entirely right?

Reply:
Done.